### ISSUE TITLE
TypeError: Cannot add property vfs...

### ISSUE THREAD

**Issue by avpengage** [BayBreezy/nuxt-pdfmake#1]
Hi there,

I'm trying to use your package with Nuxt 3.7.0 but it is throwing the following error when I try and use

```
usePDFMake.mjs?v=67cb9149:7 Uncaught (in promise) TypeError: Cannot add property vfs, object is not extensible
    at usePDFMake (usePDFMake.mjs?v=67cb9149:7:15)
```

const pdfMake = usePDFMake();

seems to stem from here:

```
export const usePDFMake = () => {
  const pdfFonts: TFontDictionary = {
    Roboto: useFontPresets().Roboto,
  };
  (<any>pdfMake).vfs = pdfFonts; // <=== Error here

  return pdfMake;
};
```

Any insight into this?  Am I doing something wrong?

Thanks for your package and any help.

Anthony

**Comment by BayBreezy** [comment]
Hello @avpengage Thanks for trying out the package.
I am trying to get this issue resolved. Can you use the plugin for now please? Until I get it resolved.

**Comment by avpengage** [comment]
Hi,

when I use:

`const { $pdfMake } = useNuxtApp();
`

things appear to be working fine.  This is fine for me.  Thanks.

I'll let you know if I have any other issues.

Thanks again.

**Comment by BayBreezy** [comment]
The issue should now be resolved with v0.0.14. Please try it out and let me know

**Comment by avpengage** [comment]
Yes.  Works now.  Nice, simple solution too :-).

Thanks!
